### PR TITLE
Allow configuration of keptn_endpoint, keptn_bridge and keptn_api_token with arguments/credential

### PR DESCRIPTION
**Pull request by augustin-dt**
Issue #12
* `keptn_endpoint` and `keptn_bridge` are now written to the `keptn.init.json` in the `initKeptn` method to allow later usage without need to set them again or use global variables.
* `keptn_api_token` can now be set in the jenkins credentials, if not present it falls back to the global variables.
* Fail the build immediately if mandatory arguments are not set.

### CODE REVIEW

File: src/sh/keptn/Keptn.groovy [hunk 1]
```diff
@@ -1,5 +1,9 @@
 package sh.keptn
 
+import org.jenkinsci.plugins.plaincredentials.StringCredentials
+import com.cloudbees.plugins.credentials.CredentialsProvider
```
Comment (grabnerandi) [inline src/sh/keptn/Keptn.groovy:4]: just double checking: are these cloudbees plugins "standard" plugins that come with a default jenkins installation? or do we add a new dependency here?

**Comment by grabnerandi** [comment]
Can we also update the readme to reflect the change? 
Would be good to include some screenshots that show how to configure the credential

**Comment by grabnerandi** [comment]
Another thing @augustin-dt  - you need to sign the DCO

**Comment by augustin-dt** [comment]
> Can we also update the readme to reflect the change?
> Would be good to include some screenshots that show how to configure the credential

Definitely, I was planning to add the readme changes to the PR after I got some feedback and nothing major must be changed

File: src/sh/keptn/Keptn.groovy (hunk 1 above)
Comment (augustin-dt) [reply src/sh/keptn/Keptn.groovy:4]: Both dependencies are from plugins that come with the default jenkins installation
* https://plugins.jenkins.io/plain-credentials/
* https://plugins.jenkins.io/credentials/

Still can't hurt to add them to the list of required jenkins plugins for the library

**Review by grabnerandi — APPROVED** [review]
looks great

**Pull request merged**